ql/experimental/inflation/cpicapfloortermpricesurface.hpp

```diff
@@ -235,14 +233,13 @@ namespace QuantLib {
 
     #endif
 
-    
-    
-    template<class I2D>
-    void InterpolatedCPICapFloorTermPriceSurface<I2D>::
-    update() {
-        notifyObservers();
-    }
-
+    namespace {
```

Comment:
We're in a header, which can be included in multiple units and defeats the purpose of an anonymous namespace.  You need to use a `namespace detail` instead.

Reply:
You are right!